Common/src/main/resources/assets/biomeswevegone/lang/ru_ru.json

```
@@ -757,25 +790,44 @@
+  "block.biomeswevegone.smooth_pink_sandstone": "Гладкий розовый песчаник",
   "block.biomeswevegone.smooth_pink_sandstone_slab": "Плита из гладкого розового песчаника",
   "block.biomeswevegone.smooth_pink_sandstone_stairs": "Ступени из гладкого розового песчаника",
-  "block.biomeswevegone.smooth_pink_sandstone": "Гладкий розовый песчаник",
+  "block.biomeswevegone.smooth_purple_sandstone": "Гладкий фиолетовый песчаник",
   "block.biomeswevegone.smooth_purple_sandstone_slab": "Плита из гладкого фиолетового песчаника",
   "block.biomeswevegone.smooth_purple_sandstone_stairs": "Ступени из гладкого фиолетового песчаника",
-  "block.biomeswevegone.smooth_purple_sandstone": "Гладкий фиолетовый песчаник",
+  "block.biomeswevegone.smooth_white_sandstone": "Гладкий белый песчаник",
   "block.biomeswevegone.smooth_white_sandstone_slab": "Плита из гладкого белого песчаника",
   "block.biomeswevegone.smooth_white_sandstone_stairs": "Ступени из гладкого белого песчаника",
-  "block.biomeswevegone.smooth_white_sandstone": "Гладкий белый песчаник",
+  "block.biomeswevegone.smooth_windswept_sandstone": "Гладкий выветренный песчаник",
   "block.biomeswevegone.smooth_windswept_sandstone_slab": "Плита из гладкого выветренного песчаника",
   "block.biomeswevegone.smooth_windswept_sandstone_stairs": "Ступени из гладкого выветренного песчаника",
-  "block.biomeswevegone.smooth_windswept_sandstone": "Гладкий выветренный песчаник",
   "block.biomeswevegone.snowdrops": "Подснежники",
+  "block.biomeswevegone.soul_fruit": "Плод душ",
+  "block.biomeswevegone.spirit_bookshelf": "Книжная полка из духовного дерева",
+  "block.biomeswevegone.spirit_button": "Кнопка из духовного дерева",
+  "block.biomeswevegone.spirit_crafting_table": "Верстак из духовного дерева",
+  "block.biomeswevegone.spirit_door": "Дверь из духовного дерева",
+  "block.biomeswevegone.spirit_fence": "Забор из духовного дерева",
+  "block.biomeswevegone.spirit_fence_gate": "Калитка из духовного дерева",
+  "block.biomeswevegone.spirit_hanging_sign": "Подвесная табличка из духовного дерева",
+  "block.biomeswevegone.spirit_leaves": "Листва духовного дерева",
+  "block.biomeswevegone.spirit_log": "Бревно из духовного дерева",
+  "block.biomeswevegone.spirit_planks": "Доски из духовного дерева",
+  "block.biomeswevegone.spirit_pressure_plate": "Нажимная плита из духовного дерева",
+  "block.biomeswevegone.spirit_roots": "Корни из духовного дерева",
+  "block.biomeswevegone.spirit_sapling": "Саженец из духовного дерева",
+  "block.biomeswevegone.spirit_sign": "Табличка из духовного дерева",
+  "block.biomeswevegone.spirit_slab": "Плита из духовного дерева",
+  "block.biomeswevegone.spirit_stairs": "Ступени из духовного дерева",
+  "block.biomeswevegone.spirit_trapdoor": "Люк из духовного дерева",
+  "block.biomeswevegone.spirit_wood": "Кора из духовного дерева",
   "block.biomeswevegone.stripped_aspen_log": "Очищенная древесина осины",
```

"Очищенные" только гифы незерской древесины, а обычная древесина "обтёсанная". Здесь вообще бревно, а не древесина.
```suggestion
  "block.biomeswevegone.stripped_aspen_log": "Обтёсанное осиновое бревно",
```
Да и всё что "из осины" должно стать "осиновым", раз уж в начале файла оно уже "осиновое".